src/Atmos/Model/tendencies.jl

```diff
@@ -0,0 +1,15 @@
+#####
+##### Tendency specification
+#####
```

Comment:
We should be very clear what we define as a tendency --- I think there is a lot of confusion between the various labels we commonly use (tendency, term, etc)

Reply:
I agree that we should have a clear and documented terminology. I've been using "tendency" to mean any RHS term, be it a flux or non-conservative source, in a spatially/temporally-continuous sense.

Reply:
This would be super helpful for me, too - as I have been a bit puzzled by this terminology to be honest, but I am not familiar with other climate codes, so I thought that it was due to my unfamiliarity

Reply:
IIUC, tendencies of `Y_j` are individual terms on the RHS of the time-stepped governing equation for `Y_j`, which have units `units(Y_j)/time`, i.e., `∂_t Y_j = Σ_i tendency_i`. Maybe others can chime in? I believe it's a temporally-continuous concept, so there shouldn't be ambiguities regarding the implicitness of the way each tendency is treated.

Reply:
If people prefer "term", I think that would be fine, but I think "tendency" is actually more specific. It not only restricts the units of the quantity, but also the context (a single term in a time-stepped equation). I was a bit skeptical of this terminology at first (we usually use RHS terms in Mech Eng), but I've come around to like it.

Reply:
Perhaps there's ambiguity with the time-dependent term and sign, however. I realize the definition I gave is vague to disambiguate `∂_t Y_j = Σ_i tendency_i` from `Σ_i tendency_{i,j} = 0`. So, we'd need a sign convention with this definition.

Reply:
I'll move the `tendencies_sum.jl` to `BalanceLaws/` and add some documentation there.

Reply:
How about both? `tendency term`? `tendency` can refer to both a `tendency_i` and `∂_t Y_j`, which might be ambiguous, and `term` can refer to any type of term in an equation. combining the two seems to be specific enough

Reply:
Thanks @charleskawczynski for the explanation. So from what I gather, these are the spatial terms, as in everything else that equals the time derivative, as you wrote above `∂_t Y_j = Σ_i tendency_i` . But you also specified RHS terms, meaning when you use an explicit time stepper. What about when you use an implicit one? Are the spatial terms still called tendencies?

Reply:
I apologize, I've abused the phrase "RHS term" a bit. My understanding is that tendencies are temporally-continuous. So, whether a term is treated explicitly / implicitly is orthogonal. I.e., we could label any tendency to be treated explicitly / implicitly in time.

Reply:
Okay, so basically I gather that is the totality of all spatial terms that equal the time variation in a conservation form, does not matter if/how discretized